src/backend/utils/page-builder.js

```diff
@@ -1,28 +1,142 @@
+ *           status: comment.status,
+ *           imgSrc: comment.imgPath,
+ *           date: moment(comment.createdAt).format('YYYY-MM-DD')
+ *         }
+ *       })
+ *     }
+ *     await this.addOverview('Collections overview', 'stats')
+ *     await this.addBlock({
+ *       title: 'The number of all articles',
+ *       value: articlesCount,
+ *       icon: 'fas fa-arrow-alt-circle-up fa-2x',
+ *       columns: 3,
+ *     }, this.colorTypes.info)
+ *     await this.addBlock({
+ *       title: 'Published articles',
+ *       value: publishedArticlesCount,
+ *       icon: 'fas fa-star fa-2x',
+ *       columns: 3
+ *     }, this.colorTypes.succes)
+ *     await this.addBlock({
+ *       title: 'Unpublished articles',
+ *       value: unpublishedArticlesCount,
+ *       icon: 'fas fa-arrow-alt-circle-down fa-2x',
+ *       columns: 3
+ *     }, this.colorTypes.warning)
+ *     await this.addBlock({
+ *       title: 'The number of users',
+ *       value: usersCount,
+ *       icon: 'fas fa-star fa-2x',
+ *       columns: 3
+ *     }, this.colorTypes.info)
+ *     mappedComments && await this.addInfoTable({
+ *       title: 'Table Information',
+ *       headers: Object.keys(mappedComments[0]),
+ *       items: mappedComments,
+ *       columns: 12
+ *     })
+ *     await this.addTextBox({
+ *       title: 'Lorem ipsum text title',
+ *       content: `
+ *         <div> lorem ipsum contentum textum boxum</div>
+ *         <div> lorem ipsum contentum textum boxum</div>
+ *         <div> lorem ipsum contentum textum boxum</div>
+ *         <div> lorem ipsum contentum textum boxum</div>
+ *       `,
+ *       columns: 6
+ *     })
+ *     mappedComments && await this.addInfoList({
+ *       title: 'Recent comments',
+ *       subtitle: 'Latest comments from user all around the world',
+ *       columns: 6,
+ *       items: mappedComments
+ *     })
+ *     await this.addTextBox({
+ *       title: 'Simple textbox',
+ *       content: '<div> lorem ipsum contentum textum boxum</div>',
+ *       columns: 6
+ *     })
+ *     await this.addChart({
+ *       columns: 6,
+ *       config: {
+ *         name: 'example',
+ *         type: 'line',
+ *         data: {
+ *           labels: clients.map(client => client.month),
+ *           datasets: [
+ *             {
+ *               label: 'label example',
+ *               fill: true,
+ *               backgroundColor: 'orange',
+ *               borderColor: 'tomato',
+ *               data: clients.map(client => client.amount),
+ *             },
+ *           ],
+ *         },
+ *         options: {
+ *           title: {
+ *             fontSize: 20,
+ *             display: true,
+ *             text: 'Title'
+ *           },
+ *         },
+ *       }
+ *     })
+ *   }    
+ * }
  */
+
 class PageBuilder {
   /**
    *
    * @param  {AdminBro}     options.admin  current instance of AdminBro
+   * @param  {String[]}     _pageContent   array of html elements as String
```

Comment:
those are params of the constructor. So only options.admin should be there. _pageContent for example is not a param - its an object property, so it should be documented like that.

Take a looka at documentation for resources property: https://github.com/SoftwareBrothers/admin-bro/blob/master/src/admin-bro.js#L86